etc/cryptor-mock/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM	node:alpine
+
+WORKDIR /src
+
+RUN npm i express body-parser
+COPY	cryptor-mock.js /src
```

Comment:
you know, I was thinking we could use the same dockerfile as the main app with a different yarn run command to spin up the cryptor mock.  No need for an extra Dockerfile.